portals/developer-portal/samples/layouts/green-theme/styles/main.css

```
@@ -0,0 +1,507 @@
+.common-btn-primary {
+  display: inline-block;
+  background-color: var(--primary-main-color);
+  color: var(--light-text-color);
+  font-size: 0.875rem;
+  padding: 0.625rem 1.25rem;
+  border: none;
+  border-radius: 0.5rem;
+  text-decoration: none;
+  cursor: pointer;
+  text-align: center;
+  transition: all 0.3s ease-in-out;
```

Stylelint (17.12.0)

[error] Duplicate property "transition" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)